src/importexport/export/qml/Export/ExportDialog.qml

```diff
@@ -55,433 +52,517 @@ StyledDialogView {
-                        }
 
-                        Loader {
-                            id: control
+                            Loader {
+                                id: control
 
-                            Layout.preferredWidth: root.smallDropdownWidth
+                                Layout.preferredWidth: root.smallDropdownWidth
 
-                            sourceComponent: {
-                                switch (type) {
-                                case ExportOptionType.TypeEnum:   return enumComp
-                                case ExportOptionType.TypeBool:   return boolComp
-                                case ExportOptionType.TypeRange:  return rangeComp
-                                case ExportOptionType.TypeString: return strComp
+                                sourceComponent: {
+                                    switch (type) {
+                                    case ExportOptionType.TypeEnum:   return enumComp
+                                    case ExportOptionType.TypeBool:   return boolComp
+                                    case ExportOptionType.TypeRange:  return rangeComp
+                                    case ExportOptionType.TypeString: return strComp
+                                    }
                                 }
-                            }
 
-                            onLoaded: if (item) item.option = option
+                                onLoaded: if (item) item.option = option
+                            }
                         }
                     }
                 }
             }
-        }
 
-        BaseSection {
-            title: qsTrc("export", "Rendering")
+            SeparatorLine {}
+
+            BaseSection {
+                id: renderingSection
+                title: qsTrc("export", "Rendering")
+
+                navigation.section: root.navigationSection
+                navigation.order: audioSection.navigation.order + 1
 
-            ColumnLayout {
-                CheckBox {
-                    id: trimBlankSpaceCheckBox
-                    width: parent.width
+                ColumnLayout {
+                    CheckBox {
+                        id: trimBlankSpaceCheckBox
+                        width: parent.width
 
-                    text: qsTrc("export", "Trim blank space before first clip")
-                    enabled: false
+                        text: qsTrc("export", "Trim blank space before first clip")
+                        enabled: false
 
-                    // navigation.name: "TrimBlankSpaceBox"
-                    // navigation.panel: root.navigation
-                    // navigation.row: 1
-                    // navigation.column: 0
+                        navigation.name: "TrimBlankSpaceBox"
+                        navigation.panel: renderingSection.navigation
+                        navigation.order: 1
+                        navigation.accessible.name: text + " checkbox"
```

Comment:
you don't need `" checkbox"` - the screen reader decides the pronunciation of the names of the types